README.yaml

```diff
@@ -0,0 +1,108 @@
+---
+#
+# This is the canonical configuration for the `README.md`
+# Run `make readme` to rebuild the `README.md`
+#
+
+# Name of this project
+name: terraform-aws-ec2-ami-backup
+
+# Logo for this project
+#logo: docs/logo.png
+
+# License of this project
+license: "APACHE2"
```

Comment:
This should have the same license as the upstream (`MIT`)

Also, add `copyrights` section with `Chef Software, Inc` with a year of `2016`